src/utils/review.ts

```diff
@@ -0,0 +1,29 @@
+import { AdminReview } from "@/services/api/admin/useReviews";
+
+function isGreaterThan24Hours(date: Date): boolean | null {
+  if (!date) return null;
+  // Get the current date and time
+  const createdReview = new Date(date);
+  const now: Date = new Date();
+  const differenceInMillis: number = now.getTime() - createdReview.getTime();
+
+  const millisecondsIn24Hours: number = 24 * 60 * 60 * 1000;
+
+  return differenceInMillis > millisecondsIn24Hours;
+}
+
+export const getReviewStatus = (review: AdminReview) => {
+  if (review.mergedAt) {
+    return "Merged";
+  } else if (
+    review.transcript.status === "not queued" &&
```

Comment:
Let's convert the status to lowercase to ensure the condition is "Javascript" safe
```suggestion
    review.transcript.status.toLowerCase() === "not queued" &&
```